### DISCUSSION TITLE
Multiple Windows

### DISCUSSION THREAD

**Discussion by Galbi3000** [OpenRCT2/OpenRCT2#16881] [Feature requests]
This is not quite the same as multiple monitor supprt but is somewhat related in a way.

I play the game in windowed mode instead of fullscreen because I use multiple monitors and it makes it easier to do things on the other monitor while playing.

I think it would be a nice feature for windows in the game to be optionally separated from the game window into their own window that can then be moved to my other monitor. Like if I want to have the large stats window open but still see what is going on in the park, I could 'undock' it to it's own window and move it to the other monitor.

Microsoft's Flight Simulator X has this sort of feature as well as a couple other games/simulators I have played in the past.

Just an idea. 🙂

**Comment by IntelOrca** [comment]
A workaround is to setup a multiplayer server and have a second client which would provide you with another window.
